.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.{kt,kts}]
+disabled_rules=import-ordering
```

Comment:
why??

Reply:
that line removes the alphabetical imports ordering in ktlint